plugin.js

```diff
@@ -145,11 +146,11 @@ function processWorkletFunction(t, fun) {
   const variables = Array.from(closure.values());
 
   const privateFunctionId = t.identifier('_f');
+  const workletID = hash(fun.toString());
```

Comment:
can we use the result of `buildWorkletString` call few lines below. I think it makes more sense that we make hash based on the actual content of `asString`

Reply:
We can but it shouldn't matter cause buildWorkletString is the same for identical functions.

Reply:
Done.